kas/iot2050.yml

```diff
@@ -10,6 +10,9 @@
 
 header:
   version: 10
+  includes:
+    - file: kas-cip.yml
+      repo: cip-core
```

Comment:
Just noticed: This is pull much more that we need and we want.

Reply:
@gylstorffq This is resolved with the latest version, right?

Reply:
@jan-kiszka Yes this is resolved by the latest version